deepmd/utils/learning_rate.py

```diff
@@ -105,3 +105,174 @@ def start_lr(self) -> float:
     def value(self, step: int) -> float:
         """Get the lr at a certain step."""
         return self.start_lr_ * np.power(self.decay_rate_, (step // self.decay_steps_))
+    
+class LearningRateCos:
+    r"""The cosine decaying learning rate.
+
+  The function returns the decayed learning rate.  It is computed as:
+  ```python
+  global_step = min(global_step, decay_steps)
+  cosine_decay = 0.5 * (1 + cos(pi * global_step / decay_steps))
+  decayed = (1 - alpha) * cosine_decay + alpha
+  decayed_learning_rate = learning_rate * decayed
+  ```
```

Comment:
Please use this style: https://numpydoc.readthedocs.io/en/latest/format.html#other-points-to-keep-in-mind